examples/OpenCL-clhpp/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download_opencl_clhpp)
+
+hunter_add_package(OpenCL)
+find_package(OpenCL CONFIG REQUIRED)
+
+hunter_add_package(OpenCL-clhpp)
+find_package(OpenCL-clhpp CONFIG REQUIRED)
+
+include_directories("${OPENCL_CLHPP_INCLUDE_DIRS}")
```

Comment:
No full paths. Since this package is cacheable (relocatable) all include paths should be propagated through imported target (like OpenCL::OpenCL).